src/index.js

```diff
@@ -0,0 +1,118 @@
+import ui from './modules/ui';
+import Gameboard from './modules/data/gameboard';
+import Player from './modules/data/players';
+
+   const playerStorage = new Array(100).fill(null);
+   const computerStorage = new Array(100).fill(null);
+
+   const playerBoard = Gameboard();
+   const computerBoard = Gameboard();
+   
+   const player = Player('player', playerBoard, playerStorage);
+   const computer = Player('computer', computerBoard, computerStorage);
+
+   let turn = 'player';
+   
+  const renderBoards = () => {
+   player.showBoard(playerStorage, 'playerBoard');
+   computer.showBoard(computerStorage, 'computerBoard');
+  };
+  
+  const computerInit = () => {
+    const { ships: [carrier, battleship, cruiser, submarine, destroyer ] } = computerBoard
+    const predefinedPlacements = [
+     [ [carrier, 0, computerStorage, 'down'], 
+       [battleship, 2, computerStorage, 'right'], 
+       [cruiser,7 , computerStorage, 'down'], 
+       [submarine, 41, computerStorage, 'right'],
+       [destroyer, 70, computerStorage, 'right'] 
+     ], 
+      [ [carrier, 0, computerStorage, 'down'], 
+      [battleship, 2, computerStorage, 'down'], 
+      [cruiser, 3, computerStorage, 'down'], 
+      [submarine, 4, computerStorage, 'down'],
+      [destroyer, 5, computerStorage, 'down'] 
+     ], 
+     [ [carrier, 9, computerStorage, 'down'], 
+     [battleship, 2, computerStorage, 'right'], 
+     [cruiser, 30, computerStorage, 'down'], 
+     [submarine, 44, computerStorage, 'right'],
+     [destroyer, 55, computerStorage, 'down'] 
+   ]
+    
+    ];
+   let randomPlacement = Math.floor(Math.random() * 3);
```

Comment:
**Follow up:** This is actually selecting a random set of predefined positions for each ship, it would be awesome to see you take care of positioning in a dynamic way.